ops/testing.py

```diff
@@ -11,8 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""Framework for unit testing charms in a simulated Juju environment.
 
-"""Infrastructure to build unit tests for charms using the ops library."""
+The module provides:
+
+- :class:`ops.testing.Harness`, a class to set up the environment for charms, and its:
```

Comment:
Perhaps "a testing harness that provides the simulated environment"? Also, ending with "and its:" seems strange to me -- perhaps "and various methods, including:"

Reply:
cleaned up.